go.mod

```diff
@@ -6,14 +6,14 @@ require (
 	github.com/go-logr/logr v1.2.3
 	github.com/onsi/ginkgo/v2 v2.6.1
 	github.com/onsi/gomega v1.24.1
-	github.com/operator-framework/api v0.14.0
+	github.com/operator-framework/api v0.17.4-0.20230223191600-0131a6301e42
 	github.com/prometheus/client_golang v1.14.0
 	github.com/prometheus/client_model v0.3.0
-	k8s.io/api v0.26.0
-	k8s.io/apimachinery v0.26.0
-	k8s.io/client-go v0.26.0
-	k8s.io/utils v0.0.0-20221128185143-99ec85e7a448
-	sigs.k8s.io/controller-runtime v0.14.1
+	k8s.io/api v0.26.1
+	k8s.io/apimachinery v0.26.1
```

Comment:
Had a Q from Lucky's PR about why not 0.26.2 for apimachinery.  Do we have a position on this we can express?

Reply:
Latest patch releases are always better, since K8s 1.26.2 does fix a few [regressions](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.26.md#changelog-since-v1261). Though it mayn't affect anything here directly, its better to use the latest ones to be on the safe side.

Reply:
Makes sense. That sounds good to me! I was a bit hesitant as I thought we should be keeping it consistent across all the repos, but if that's not the case, I agree with your point. 

Bumped k8s to latest patch release `v0.26.2`